src/include/miopen/find_solution.hpp

```diff
@@ -170,11 +170,7 @@ struct SolverContainer
                     {
                         /// \todo If Solver is applicable it must provide an appropriate Solution.
                         /// This is not the case for some 20x5 convolutions (and possibly others).
-                        /// Normally we should not get here and message level should be Error.
-                        /// For now, let's use Info (not Warning) level to avoid
-                        /// flooding the console.
-                        MIOPEN_LOG_I(SolverDbId(solver)
-                                     << ": [Warning] Applicable Solver not succeeded.");
+                        MIOPEN_LOG_E(SolverDbId(solver) << ": Applicable Solver not succeeded.");
```

Comment:
Please revert this. https://github.com/ROCmSoftwarePlatform/MIOpen/pull/910/files#r657108163 is not related to non-convolution primitives only.